### PR TITLE
Scoreboard fixes

### PR DESCRIPTION
**Pull request by TerminalHash**
## Status
- **READY**: this PR is (to the best of your knowledge) ready to be incorporated into the game.

## Description
Map names, team names on ru language and match timer broken after this commit: a32a46182dfc57682b830ecab12b30dcbc3f55a3

This pull request fixing this.

## Steps to Test or Reproduce
Join on vanilla CTF or host local dedicated server with CTF mode, use locale RU and open scoreboard.

### CODE REVIEW

File: Rules/CommonScripts/ScoreboardCommon.as [hunk 1]
```diff
@@ -138,6 +137,40 @@ string timestamp(uint s)
...
 }
 
+void getMapName(CRules@ this)
+{
+	CMap@ map = getMap();
+
+	if (map !is null && isServer())
+	{
+		string[] name = map.getMapName().split('/');	 //Official server maps seem to show up as
+		string mapName = name[name.length() - 1];		 //``Maps/CTF/MapNameHere.png`` while using this instead of just the .png
```
Comment (Vam-Jam) [inline Rules/CommonScripts/ScoreboardCommon.as:147]: Indent

File: Rules/CommonScripts/ScoreboardRender.as [hunk 2]
```diff
@@ -90,7 +90,12 @@ float drawScoreboard(CPlayer@ localplayer, CPlayer@[] players, Vec2f topleft, CT
 	topleft.y += stepheight;
 
 	//draw team info
-	GUI::SetFont("AveriaSerif-Bold_22");
+	if (g_locale == "ru") { // because averia dont have cyrillic symbols, we need use another font
```
Comment (Vam-Jam) [inline Rules/CommonScripts/ScoreboardRender.as:93]: Move the comment above the statement

**Review by Vam-Jam — CHANGES_REQUESTED** [review]
Before:


After:


Match timer now only increments after the game has started, which seems to be how the script should have worked before it was broken.

lgtm after the comments are fixed

File: Rules/CommonScripts/ScoreboardCommon.as (hunk 1 above)
Comment (TerminalHash) [reply Rules/CommonScripts/ScoreboardCommon.as]: fixed

File: Rules/CommonScripts/ScoreboardRender.as (hunk 2 above)
Comment (TerminalHash) [reply Rules/CommonScripts/ScoreboardRender.as]: moved

**Review by Vam-Jam — APPROVED** [review]

**Pull request merged**